### ISSUE TITLE
When will LLAMA2 Mplug-owl be released? 

### ISSUE THREAD

**Issue by heylamourding** [X-PLUG/mPLUG-Owl#146]
Hi team, 

May I know when will LLAMA2 based mplug-owl be released?

**Comment by MAGAer13** [comment]
Scheduled in next month.

**Comment by xmy0916** [comment]
@MAGAer13 快10月咯！月底还能等得到吗！期待

**Comment by MAGAer13** [comment]
We will release mPLUG-Owl-2 recently, which is the next generation of mPLUG-Owl. As well as the pre-trained weight / paper.

**Comment by AnnaYang2020** [comment]
> We will release mPLUG-Owl-2 recently, which is the next generation of mPLUG-Owl. As well as the pre-trained weight / paper.

Is the "mplug-2" you recently released the "mplug-owl-2B" in quote?

**Comment by MAGAer13** [comment]
> > We will release mPLUG-Owl-2 recently, which is the next generation of mPLUG-Owl. As well as the pre-trained weight / paper.
> 
> Is the "mplug-2" you recently released the "mplug-owl-2" in quote?

No, it is mPLUG-Owl2, the next generation of mPLUG-Owl. About 8.3B parameters
